src/EA.Weee.Web/Areas/Admin/Controllers/AddAatfController.cs

```diff
@@ -135,6 +135,7 @@ private AatfData CreateAatfData(AddAatfViewModel viewModel)
                 Guid.NewGuid(),
                 viewModel.AatfName,
                 viewModel.ApprovalNumber,
+                null,
```

Comment:
It is possible to remove the operator parameter completely from the constructor? not sure its needed anymore